wof/wofpy_config.py

```diff
@@ -14,22 +16,25 @@
 
 Usage:
     wofpy_config INDIR
+    wofpy_config INDIR [--mode=<test,production>]
 
     wofpy_config (-h | --help | -v | --version)
 
 Examples:
-    wofpy_config my_wofpy_config
+    wofpy_config wofpyserver
```

Comment:
> The script also will fail if that directory already exists. We can add an option o overwrite existing directories later.

Actually, we should accommodate the case of a user running wofpy_config on a **base** directory that already exists, to add another DAO. It should not fail or overwrite the base directory; failure or overwriting could happen for the DAO directory if it exists, though (eg odm2timeseries). See [my earlier comments](https://github.com/ODM2/WOFpy/issues/132#issuecomment-304333740)